Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -911,6 +911,10 @@ def run(self):
 
         if additionalArguments:
             self.command += additionalArguments
+            # Double the timeout for site isolation queues.
+            # FIXME: Expose the timeout property in config.json.
```

Comment:
What would actually be the better fix is https://bugs.webkit.org/show_bug.cgi?id=290867, so we don't need to rely on setting huge timeouts for each individual test. Feel free to include this in the `FIXME` but r+ nonetheless!